.github/workflows/documentation.yml

```diff
@@ -20,15 +20,26 @@ jobs:
       - name: install dependencies
         run: |
           pip install -r requirements/development.txt
-          sudo apt -y update
-          sudo apt -y install doxygen graphviz
+          sudo apt-get -y update
+          sudo apt-get -y install graphviz libclang1-9 libclang-cpp9
```

Comment:
I assume clang is needed to build doxygen? Or is it for something else?

Reply:
The doxygen binaries available from source forge are statically linked to a couple clang libs. That's why we have to install the clang stuff.

Building doxygen from source requires a lot more dependency installs and it takes longer.